src/bindings/config.ts

```diff
@@ -68,6 +74,9 @@ export const loadConfig = async (context: Context): Promise<BotConfig> => {
       privateKey: process.env.X25519_PRIVATE_KEY ?? "",
       publicKey: publicKey ?? "",
     },
+    wallet: {
+      registerWalletWithVerification: process.env.REGISTER_WALLET_WITH_VERIFICATION?.toLowerCase() === "true" ?? DEFAULT_REGISTER_WALLET_WITH_VERIFICATION,
```

Comment:
We should make the `registerWalletWithVerification` as an org/repo-specific variable, not the bot-specific variable.

What needs to be updated for that?
1/ Read the value from `getWideConfig` and set the value like `baseMultiplier`/`issueCreatorMultiplier`.
2/ Remove `DEFAULT_REGISTER_WALLET_WITH_VERIFICATION`

Reply:
This is done and here is a new QA after making this fix https://github.com/web4er/ubiquibot/issues/25